tests/test_validators.py

```diff
@@ -73,6 +73,14 @@ def test_s3_bucket_name(self):
             with self.assertRaises(ValueError):
                 s3_bucket_name(b)
 
+    def test_elb_name(self):
+        for b in ['a', 'a'*32, 'wick3d-elb-name', 'Wick3d-ELB-Name']:
+            elb_name(b)
+        for b in ['a'*63, 'invalid_elb', '-invalid-elb',
```

Comment:
Good catch.  Copy pasta.  Fixed and expanded in 9b55342afe7a2f14cb126f49441fb19a8d514470